prebuilt-ui/basic-embed/components/PrebuiltCall.js

```diff
@@ -0,0 +1,254 @@
+import DailyIframe from '@daily-co/daily-js';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { writeText } from 'clipboard-polyfill';
+import { Button } from '@dailyjs/shared/components/Button';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+} from '@dailyjs/shared/components/Card';
+import Field from '@dailyjs/shared/components/Field';
+import { TextInput } from '@dailyjs/shared/components/Input';
+import { Well } from '@dailyjs/shared/components/Well';
+import { Header } from '../components/Header';
+
+export default function PrebuiltCall(props) {
+  const [demoState, setDemoState] = useState('home');
+  const [isError, setIsError] = useState(false);
+  const [roomURL, setRoomURL] = useState('');
+  const [exp, setExp] = useState();
+  const [secs, setSecs] = useState();
+  const [roomValidity, setRoomValidity] = useState(false);
+  const roomURLRef = useRef(null);
+  const iframeRef = useRef(null);
+  const callFrame = useRef(null);
+  const [linkCopied, setLinkCopied] = useState(false);
```

Comment:
small nit: keep state declarations together

Reply:
I usually put refs at the top but totally up to you

Reply:
Oh, thank you! I feel like I had something set up when I was working on docs that did this for me 🤔